app/controllers/flood_risk_engine/enrollments/addresses_controller.rb

```diff
@@ -0,0 +1,107 @@
+require_dependency "flood_risk_engine/application_controller"
+
+module FloodRiskEngine
+  module Enrollments
+    class AddressesController < ApplicationController
```

Comment:
what do you mean by 'normal' - surely it's all one workflow ? I would have thought this is just a side branch with a couple of available back transitions.